src/__tests__/internal/Panel.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import Panel from '../../components/internal/Panel';
+
+// Shallow Rendering
+// https://github.com/airbnb/enzyme/blob/master/docs/api/shallow.md
+describe('Panel - Shallow Rendering', () => {
+    it('to have 1 Panel class', () => {
+      const wrapper = shallow(
+        <Panel
+          id="Panel"
+          text="PanelPanelPanel"
+          level="level 1"
+        />,
+      );
+      expect(wrapper.find('.mars')).to.have.length(1);
+    });
+});
+
```

Comment:
Add updated code for the panel, the new code has the option to add a 'close' button for showing errors etc.